active/package-maintainers.md

```diff
@@ -0,0 +1,126 @@
+
+## Group communications
+
+The group communicates asynchronously via:
+
+- The public `#packages` channel on the Django Discord server.
+- Appropriate public sections of the [Django Forum](https://forum.djangoproject.com/).
+- GitHub issues and pull requests in a new `django/package-maintainers-wg` repository.
+- Collaborative documents set up in the Django Software Foundation’s Google Workspace.
+- A new private #package-maintainers-wg` channel on the Django Discord server for internal discussions.
+
+The group will also offer office hours for package maintainers every month via a video call.
+
+## Reporting
+
+The group will share highlights from current activities on a monthly basis via a forum post.
+
+## Appendix
+
+### Group activities
+
+As an illustration of the group’s remit, here are possible activities members could take part in.
+
+#### Support Django version compatibility efforts
+
+As Django regularly releases new versions, it’s an ongoing effort to ensure packages are compatible with new releases.
+
+Group members can support maintainers in this effort by:
+
+- Creating a shared calendar with Django release dates and deadlines.
+- Creating communication channels for package maintainers to coordinate compatibility changes for a specific release.
+- Recommending appropriate automation, such as [django-upgrade](https://github.com/adamchainz/django-upgrade) or how to set up Django pre-releases in Continuous Integration.
+- Proposing additions to the Django release notes with recommendations on how to handle deprecations.
+
+#### Share packaging best practices
+
+The group can work on create, maintain, or curate best practices for Django package maintainers. For example, the official [Advanced tutorial: How to write reusable apps](https://docs.djangoproject.com/en/5.0/intro/reusable-apps/), or packaging-focused sections on third-party resources like [awesome-django](https://github.com/wsvincent/awesome-django).
+
+Those best practices can cover aspects like:
+
+- Documentation approaches and tools
+- Use of the `django_` prefix for app names (see [Change reusable apps naming recommendation](https://forum.djangoproject.com/t/change-reusable-apps-naming-recommendation/25233)).
+- Recommended Python, Django, browser, operating system support policies to align with those of Django itself.
+- Tooling, for example how to set up Continuous Integration for Django packages.
+
+#### Facilitate community-driven maintenance
+
+Group members can liaise with organizations dedicated to sharing maintenance efforts, such as [Jazzband](https://jazzband.co/), [Wagtail Nest](https://github.com/wagtail-nest), or [Django Commons](https://github.com/django-commons). Or work directly with packages in need of new maintainers to find new candidates. This can involve:
+
+- Sharing calls for new package maintainers
+- Coordinating trusted volunteer open source “roadies” familiar with package ownership transfers.
```

Comment:
In case it's helpful, I [wrote a bit recently](https://harihareswara.net/posts/2024/trust-new-maintainer/) about some ways to check for trustworthiness in volunteer candidates for maintainership. A WG could do more than an individual maintainer could in more thoroughly checking candidates, and that could make an individual maintainer feel more okay about letting a vetted volunteer co-maintain their package.